### DISCUSSION TITLE
Cara mengatur jumlah angka desimal

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#408] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/197515189-af43d4a4-0296-4b9d-8314-2d282eeb16b8.png)
ijin bertanya bagaimana cara mengatur jumlah angka desimal (di belakang koma/titik)

**Comment by VictorNugraha** [comment, marked as answer]
hai, @19807311!

Untuk mengatur koma pada hasil tooltip yang ditampilkan, Ibu bisa menggunakan fungsi **comma()** dari **library(scales)**. Pada fungsi tersebut dapat di-isi dengan 2 parameter yaitu, 

- **x**: Dapat di-isi dengan nama kolom numerik yang ingin diatur 
- **accuracy**: Dapat di-isi dengan angka numerik yang mengindikasikan berapa banyak angka desimal yang ingin ditunjukan. Sebagai contoh 2 angka desimal, parameter ini dapat di-isi dengan **0.01**.

Sebagai contoh, jika ingin di-implemetnasikan kedalam fungsi glue()

`mutate(label = glue("kolom_numerik = {comma(kolom_numerik, accuracy = 0.01}"))`

Demikian informasi yang dapat saya sampaikan, selamat mencoba dan semoga bermanfaat!

**Reply by 19807311** [reply]
Thank you
